subxt/src/tx/tx_progress.rs

```diff
@@ -249,14 +255,10 @@ impl<T: Config, C: Clone> Stream for TxProgress<T, C> {
 /// - FinalityTimeout
 /// - Invalid
 /// - Dropped
+///
 /// In any of these cases the client side TxProgress stream is also closed.
-/// So there is currently no way for you to tell if an Dropped`/`Invalid`/`Usurped` transaction
-/// reappears in the pool again or not.
-/// You are free to unsubscribe from notifications at any point.
-/// The first one will be emitted when the block in which the transaction was included gets
-/// finalized. The `FinalityTimeout` event will be emitted when the block did not reach finality
-/// within 512 blocks. This either indicates that finality is not available for your chain,
-/// or that finality gadget is lagging behind.
+/// In those cases you have to re-subscribe to the extrinsic/create a new TxProgress repeatedly until the extrinsic is finalized.
```

Comment:
```suggestion
    /// In those cases the stream is closed however, so you currently have no way to find 
    /// out if they finally made it into a block or not.
```

Just to revert this since turns out you can't re-subscribe :)